<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Match Maker</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="inputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0">
        <widget class="QWidget" name="inputSpacer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="databaseInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-- None --</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="databaseTitle">
         <property name="text">
          <string>Database:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="databaseQuestions">
         <property name="text">
          <string>0 questions</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="matchAmountTitle">
         <property name="toolTip">
          <string>The number of matches that will be generated for each user.</string>
         </property>
         <property name="text">
          <string>Number of Top Matches to Calculate:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="selectDatabase">
         <property name="text">
          <string>Select Database</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QSpinBox" name="matchAmountSpinner">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QDoubleSpinBox" name="rangeSpinBox">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>4.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="rangeLabel">
         <property name="toolTip">
          <string>The numerical range that a user's answers can be.
Only used for scaling the match percentages.
For example, if answers could be from 1 to 5, then the range would be 5 - 1 = 4.</string>
         </property>
         <property name="text">
          <string>Answer Range:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="progressGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Progress</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <layout class="QGridLayout" name="progressLayout">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="printingLabel">
           <property name="text">
            <string>Printing:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="femaleInfo">
           <property name="text">
            <string>0/0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="maleLabel">
           <property name="text">
            <string>Male Matching:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="totalInfo">
           <property name="text">
            <string>0/0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QProgressBar" name="maleProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QProgressBar" name="usersProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="maleInfo">
           <property name="text">
            <string>0/0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="totalLabel">
           <property name="text">
            <string>Total Matching:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="femaleLabel">
           <property name="text">
            <string>Female Matching:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="usersLabel">
           <property name="text">
            <string>Loading Users:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="usersInfo">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QProgressBar" name="femaleProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="4">
          <widget class="QProgressBar" name="totalProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="4">
          <widget class="QProgressBar" name="printingProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="printingInfo">
           <property name="text">
            <string>0/0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QWidget" name="progressSpacer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="clear">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="outputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="pdfTitle">
         <property name="text">
          <string>Print to PDF:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pdfInfo">
         <property name="text">
          <string>-- None --</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="titleTitle">
         <property name="toolTip">
          <string>What will be displayed at the top of each user's page of matches.</string>
         </property>
         <property name="text">
          <string>Page Title:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="titleEdit">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MatchMaker Matches</string>
         </property>
         <property name="cursorPosition">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="titleFontGroup">
         <property name="toolTip">
          <string>The font that will be used to display the title on each user's matches</string>
         </property>
         <property name="title">
          <string>Title Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" colspan="2">
           <widget class="QFontComboBox" name="titleFontComboBox">
            <property name="currentFont">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="titleFontSizeSpinner">
            <property name="toolTip">
             <string>Point Size</string>
            </property>
            <property name="suffix">
             <string> pt</string>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>72</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="titleFontStyleFrame">
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="titleFontStyleBold">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Bold</string>
               </property>
               <property name="text">
                <string>B</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Line" name="titleFontStyleLine1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="titleFontStyleItalic">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <italic>true</italic>
                </font>
               </property>
               <property name="toolTip">
                <string>Italic</string>
               </property>
               <property name="text">
                <string>I</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="Line" name="titleFontStyleLine2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="titleFontStyleUnderscore">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <underline>true</underline>
                </font>
               </property>
               <property name="toolTip">
                <string>Underline</string>
               </property>
               <property name="text">
                <string>U</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="matchesFontGroup">
         <property name="toolTip">
          <string>The font that will be used to display each user's matches</string>
         </property>
         <property name="title">
          <string>Matches Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0" colspan="2">
           <widget class="QFontComboBox" name="matchesFontComboBox">
            <property name="currentFont">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="matchesFontSizeSpinner">
            <property name="toolTip">
             <string>Point Size</string>
            </property>
            <property name="suffix">
             <string> pt</string>
            </property>
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>72</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="matchesFontStyleFrame">
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="matchesFontStyleBold">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="toolTip">
                <string>Bold</string>
               </property>
               <property name="text">
                <string>B</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Line" name="matchesFontStyleLine1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="matchesFontStyleItalic">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <italic>true</italic>
                </font>
               </property>
               <property name="toolTip">
                <string>Italic</string>
               </property>
               <property name="text">
                <string>I</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="Line" name="matchesFontStyleLine2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="matchesFontStyleUnderscore">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <underline>true</underline>
                </font>
               </property>
               <property name="toolTip">
                <string>Underline</string>
               </property>
               <property name="text">
                <string>U</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QWidget" name="outputSpacer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="selectPdf">
         <property name="text">
          <string>Select PDF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>selectDatabase</tabstop>
  <tabstop>start</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
